Source/Scene/Model.js

```diff
@@ -1882,15 +1886,28 @@ define([
     function modifyShaderForQuantizedAttributes(shader, programName, model) {
         var primitive;
         var primitives = model._programPrimitives[programName];
+
+        if (!defined(primitives)) {
+            return shader;
+        }
```

Comment:
What is this check for?

Reply:
If the program primitives weren't cached, they are not quantized, and the shader does not need to be modified.

Reply:
I'll add a comment if unclear.